### PR TITLE
lazy load images

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Lazy load all images.
- Thoughts???

# Needed By (Date)
- :man_shrugging: 

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear cache
1. view a page with images
1. validate the images still show when they are in the viewport.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by sherakama — APPROVED** [review]
With everything being lazy, there is potential for images to not load up and take up the correct space in the dom, so when they do, you get "jumping" content and shifting pages. Are you sure you want to load everything by lazy and do the majority of images have the correct height/width properties rendered?

File: docroot/themes/humsci/su_humsci_theme/js/su_humsci_theme.js [hunk 1]
```diff
@@ -44,7 +44,9 @@
         $(figure).imagesLoaded(function () {
           $(figure).find('figcaption, picture').css('max-width', $(figure).find('img').width());
         });
-      })
+      });
+
+      $("img.lazy").lazyload();
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_theme/js/su_humsci_theme.js]:
```suggestion
      $("img.lazy", context).lazyload();
```

File: docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme [hunk 2]
```diff
@@ -11,6 +11,19 @@ use Drupal\node\Entity\Node;
 use Drupal\Component\Utility\SortArray;
 use Drupal\Core\Template\Attribute;
 
+/**
+ * Implements hook_preprocess_HOOK.
+ *
+ * Use lazy loading on images.
+ */
+function su_humsci_theme_preprocess_image(&$variables) {
+  if (isset($variables['attributes']['width']) && isset($variables['attributes']['height'])) {
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme:20]: 👍

**Comment by sherakama** [comment]
GTG

**Comment by boznik** [comment]
have you tested page load improvements?

**Comment by pookmish** [comment]
I haven't done benchmark test, but i reviewed the network activity in the browser and saw the delay in image loads. if we get this up to staging, i'll run some tests on stage vs prod

**Review by boznik — APPROVED** [review]
This is working as expected

**Pull request merged**